console/src/components/Services/Data/Schema/Schema.js

```diff
@@ -96,7 +98,10 @@ class Schema extends Component {
 
       const _untrackedTables = schema.filter(
         table => !table.is_table_tracked && table.table_schema === currentSchema
-      );
+      ).map(table => {
```

Comment:
this is very confusing to read if reader doesn't have any context. Why is there a random `.map` to set some `schemeWarning` while getting untracked tables? 

Doing this on next line with comment makes it much clearer, 

something like:

```js
const _untrackedTables = schema.filter(
    table => !table.is_table_tracked && table.table_schema === currentSchema
);

// update with graphql compatibility
 _untrackedTables.forEach(t => {
    t.isGQLCompatible = gqlPattern.test(t.table_name);
});
```

Also rename  `schemeWarning` to `isGQLCompatible` for clarity